falcosidekick/templates/deployment.yaml

```diff
@@ -84,8 +84,31 @@ spec:
               value: {{ .Values.config.customfields | quote }}
             - name: TEMPLATEDFIELDS
               value: {{ .Values.config.templatedfields | quote }}
+            - name: BRACKETREPLACER
+              value: {{ .Values.config.bracketreplacer | quote }}
             - name: MUTUALTLSFILESPATH
               value: {{ .Values.config.mutualtlsfilespath | quote }}
+            - name: MUTUALTLSCLIENT_CERTFILE
+              value: {{ .Values.config.mutualtlsclient.certfile | quote }}
+            - name: MUTUALTLSCLIENT_KEYFILE
+              value: {{ .Values.config.mutualtlsclient.keyfile | quote }}
+            - name: MUTUALTLSCLIENT_CACERTFILE
+              value: {{ .Values.config.mutualtlsclient.cacertfile | quote }}
+            - name: TLSSERVER_DEPLOY
+              value: {{ .Values.config.tlsserver.deploy | quote }}
+            - name: TLSSERVER_CERTFILE
+              value: {{ .Values.config.tlsserver.certfile | quote }}
+            - name: TLSSERVER_KEYFILE
+              value: {{ .Values.config.tlsserver.keyfile | quote }}
+            - name: TLSSERVER_CACERTFILE
+              value: {{ .Values.config.tlsserver.cacertfile | quote }}
```

Comment:
We need to go further, as nothing is already there to add the cert and key files, these settings are useless. We need to mount secrets as volumes.

Reply:
Does including TLS and mTLS server configs kinda a repetition of the ingress job ?

With the way of doing thing inside Kubernetes, do we need to let the possibility to activate this kind of feature inside the pod ?

Reply:
I think the best method to handle the TLS and mTLS use cases, for the chart, is to use the Ingress options and not let these 2 append in the pod.

Reply:
Some users expect the pods to communicate with TLS, but OK. Let's keep as is for now.